tests/hopp/test_steel/test_eaf_model.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+from hopp.simulation.technologies.steel.eaf_model import eaf_model
```

Comment:
Please add comments to each test regarding where the values came from or how they were determined

Reply:
Added in doc strings in test files